hms-app/src/components/input-forms/HackathonForm.tsx

```diff
@@ -25,9 +25,12 @@ function HackathonForm(props: IProps) {
   const [endDateValue, setEndDateValue] = useState<Date | null>(new Date())
   const [hackathonTitle, setHackathonTitle] = useState('')
   const [hackathonSlug, setHackathonSlug] = useState('')
-  const [DescriptionValue, onChange] = useState('')
   const [hackathon, setHackathon] = useState<Hackathon>({} as Hackathon)
 
+  const initialValue =
+    '<p>Please add your <b>hackathon description</b> here</p>'
+  const [descriptionValue, onChange] = useState(initialValue)
```

Comment:
I don't think `onChange` is a good name for this.  Basically react creates a function with that name under the hood, a function named simply `onChange` doesn't provide enough context as to what it is doing.  `setDescriptionValue` I think would be the best practice here